packages/cli/src/config/schema.ts

```diff
@@ -1012,6 +1016,53 @@ export const schema = {
 		},
 	},
 
+	eventBus: {
+		enabled: {
+			doc: 'Whether event logging is enabled.',
+			format: Boolean,
+			default: true,
+			env: 'N8N_EVENTBUS_ENABLED',
+		},
+		checkUnsentInterval: {
```

Comment:
What are the pros and cons of this behavior? Are we documenting this somewhere? (A docs ticket or something to assist in deciding when to turn this on)

Reply:
agreed - there isn't much use to turn it off, but possible side effects if it is _not_ running. so I'll remove this option (a leftover from earlier iterations...)